kloppy/infra/serializers/event/wyscout/deserializer_v3.py

```diff
@@ -672,6 +672,75 @@ def _parse_period_id(raw_period: str) -> int:
     return period_id
 
 
+def insert_substitution_events(
```

Comment:
I think I would split this up into `_parse_substitutions` and the code to merge the events.